tests/classy_jest/_support/mockApiResponse.ts

```diff
@@ -0,0 +1,106 @@
+import { GetTicketApiResponse } from '@tec/tickets/classy/types/Api';
+
+/**
+ * Mock API response for ticket operations.
+ *
+ * @return {GetTicketApiResponse} The mock API response.
+ */
+export function makeMockApiResponse(): GetTicketApiResponse {
+	return {
+		id: 1,
+		date: '2024-01-01T12:00:00+00:00',
+		date_gmt: '2024-01-01T12:00:00Z',
+		guid: {
+			rendered: 'https://example.com/?p=1',
```

Comment:
Here and in the `permalink_template` I would update the fields to look more in line with what a ticket would look like, e.g. `https://example.com/tickets/ticket-1` or something along those lines.